mima.sbt

```diff
@@ -0,0 +1,181 @@
+      exclude[NewMixinForwarderProblem]("cats.kernel.Band.combineN"),
+      exclude[NewMixinForwarderProblem]("cats.kernel.Band#mcF#sp.combineN"),
+      exclude[NewMixinForwarderProblem]("cats.data.Tuple2KApply.product"),
+      exclude[NewMixinForwarderProblem]("cats.InvariantInstances0.catsApplicativeForArrow")
+    ) ++ // Additional methods in package-private traits
+    Seq(
+      exclude[ReversedMissingMethodProblem]("cats.data.NonEmptyCollection.grouped")
+    ) ++ // https://github.com/typelevel/cats/pull/3785
+    Seq(
+      exclude[MissingClassProblem]("cats.syntax.EqOps$mcJ$sp"),
+      exclude[MissingClassProblem]("cats.syntax.EqOps$mcD$sp"),
+      exclude[FinalClassProblem]("cats.syntax.EqOps"),
+      exclude[MissingFieldProblem]("cats.syntax.EqOps.lhs"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.unapply"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.apply"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.lhs"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy$default$1"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productPrefix"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productArity"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productElement"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productIterator"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.canEqual"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy$default$1$mcD$sp"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy$default$1$mcF$sp"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy$default$1$mcJ$sp"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.copy$default$1$mcI$sp"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productElementNames"),
+      exclude[DirectMissingMethodProblem]("cats.syntax.EqOps.productElementName"),
+      exclude[MissingClassProblem]("cats.syntax.EqOps$"),
+      exclude[MissingClassProblem]("cats.syntax.EqOps$mcF$sp"),
+      exclude[MissingClassProblem]("cats.syntax.EqOps$mcI$sp")
+    ) ++ // https://github.com/typelevel/cats/pull/3918
+    Seq(
+      exclude[MissingClassProblem]("algebra.laws.IsSerializable"),
+      exclude[MissingClassProblem]("algebra.laws.IsSerializable$")
+    ) ++ // https://github.com/typelevel/cats/pull/3987
+    Seq(
+      exclude[DirectAbstractMethodProblem]("cats.free.ContravariantCoyoneda.k"),
+      exclude[ReversedAbstractMethodProblem]("cats.free.ContravariantCoyoneda.k"),
+      exclude[DirectAbstractMethodProblem]("cats.free.Coyoneda.k"),
+      exclude[ReversedAbstractMethodProblem]("cats.free.Coyoneda.k")
+    )
+}
+
+// TODO: remove these filters after MiMa 1.1.0 is released with improved support for Scala 3
+ThisBuild / mimaBinaryIssueFilters ++= {
```

Comment:
@dwijnand has very kindly fixed this in https://github.com/lightbend/mima/pull/683 :)

Reply:
Sorry I've not cut a release yet - life things shuffled my priorities the last 3 weeks.

Reply:
No worries, just means we'll appreciate it even more when it arrives! :)